app/assets/stylesheets/abstracts/_layout.scss

```diff
@@ -0,0 +1,11 @@
+.d-flex {
+  display: flex;
+}
+
+.hidden {
```

Comment:
I think you can use Tailwind for these kind of utility classes cc @brunvez

Reply:
Good idea. This repo is a little bit behind the base, We haven't added tailwind yet